docs/tools/HUSKY.md

```diff
@@ -18,46 +18,19 @@ It runs on staged files via `lint-staged`. As this is a clean project it is quit
 
 ## Setup
 
-- Add workspace reference to `husky`:
-
-  ```sh
-  pnpm add -w husky -D
-  ```
-
-- Install `husky`:
-
-  ```sh
-  pnpm husky install
-  ```
-
-- Add husky's install script:
-
-  ```jsonc
-  // package.json
-
-  "scripts": {
-    ...
-    "prepare": "husky install"
-    ...
-  }
-  ```
-
 - Add `pre-commit` hook for linting and formatting indexed files using [lint-staged](./lint-staged.md):
 
   ```sh
-  pnpm husky add .husky/pre-commit 'pnpm lint-staged --quiet'
+  npx husky add .husky/pre-commit 'run-command-on-pre-commit'
```

Comment:
Is this working and only running on `staged` files? I set it up for staged only so it's not too slow running on all files

Reply:
I've just changed it to a more general term, like `run-command-on-pre-commit` is any command you want to add to a pre-commit hook.

Reply:
OK cool. I think the `lint-staged` package is removed now. That helps it only consider staged files. I can check it and re-added if needed, don't worry!